aptos-move/framework/src/natives/object.rs

```diff
@@ -25,6 +27,8 @@ use std::collections::VecDeque;
 #[derive(Clone, Debug)]
 pub struct ExistsAtGasParameters {
     pub base_cost: InternalGas,
+    pub per_byte: InternalGasPerByte,
+    pub per_item: InternalGas,
 }
 
 fn native_exists_at(
```

Comment:
why do we need this at the first place? what's missing in built-in operator `exists<T>(address): bool`